### PR TITLE
Product Updates page: design update

### PR DESCRIPTION
**Pull request by muhajirdev**
### Summary

Before
Screen Shot 2021-12-01 at 15 49 17

After
Screen Shot 2021-12-01 at 15 49 00

### CODE REVIEW

**Comment by mmahalwy** [comment]
#### 📝 Changed routes:


| [**…dates/changelog.module**](https://quran-com-p1rstqhtp-qurancom.vercel.app/product-updates/changelog.module) |
|:-:|
|<a href="https://quran-com-p1rstqhtp-qurancom.vercel.app/product-updates/changelog.module">Screenshot of /product-updates/changelog.module</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/I7W2jMAXU8GgEaZhTxMNHe9yVhz_44EivnJJ0YYJxOR-dIlzxjKQG6LTok-RcCT3e00hqrKKoGacmC694w1wGg.png">(view full size)</a> |







Commit d28ec6c6137330f268fd685985bec8568164338c (https://quran-com-p1rstqhtp-qurancom.vercel.app).

File: src/pages/product-updates/changelog.module.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
   width: 95%;
   margin: 0 auto;
   @include breakpoints.tablet {
-    width: 50%;
+    max-width: calc(35 * var(--spacing-mega));
```
Comment (osamasayed) [inline src/pages/product-updates/changelog.module.scss:8]: nit: move to a const

File: src/components/Notion/Page/Page.module.scss [hunk 2]
```diff
@@ -1,23 +1,45 @@
...
+    opacity: 1;
+  }
+  font-weight: var(--font-weight-semibold);
+  font-size: calc(0.85 * var(--font-size-normal));
+  letter-spacing: 0.5px;
+  text-transform: uppercase;
+}
+
+.blocksContainer {
+  max-width: calc(25 * var(--spacing-mega));
```
Comment (osamasayed) [inline src/components/Notion/Page/Page.module.scss:44]: nit: move to a const

**Review by osamasayed — APPROVED** [review]

**Comment by mmahalwy** [comment]
#### 📝 Changed routes:


| [**/product-updates/\[id\]**](https://quran-com-610ouis7w-qurancom.vercel.app/product-updates/[id]) | [**…dates/changelog.module**](https://quran-com-610ouis7w-qurancom.vercel.app/product-updates/changelog.module) |
|:-:|:-:|
|<a href="https://quran-com-610ouis7w-qurancom.vercel.app/product-updates/[id]">Screenshot of /product-updates/[id]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/NLNu-sK3KfnR0CXgEKJQcKU1mCsvA_0o8wipATBy8CixGzg7XlYbr11UHf8N2NEAMhXA18KjSiH7xMRcBKBt0w.png">(view full size)</a> |<a href="https://quran-com-610ouis7w-qurancom.vercel.app/product-updates/changelog.module">Screenshot of /product-updates/changelog.module</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/5ePRoRPGLjHVRub8nTDnfSpQW8SZmZJtx6frys7RieABOOArXmYtdF0BIcsYjHWRGNVDwRJ4PCeflpPHdZAdiQ.png">(view full size)</a> |







Commit b420b3cff3bf02b82a5b62833e5d332474b2c790 (https://quran-com-610ouis7w-qurancom.vercel.app).

**Pull request merged**